src/main/java/duke/DukeNoArgumentException.java

```diff
@@ -0,0 +1,11 @@
+package duke;
+
+public class DukeNoArgumentException extends DukeException {
+
+    public DukeNoArgumentException() {
+
+    }
+    public DukeNoArgumentException(String message) {
+        super(message);
```

Comment:
Here, use the opportunity provided by inheritance to include a more expressive error message rather than just invoke the super constructor.